packages/webpacker/package.json

```
@@ -39,6 +38,7 @@
     "postcss-preset-env": "^7.1.0",
     "postcss-scss": "^4.0.2",
     "sass-embedded": "^1.58.3",
+    "shakapacker": "~6.6",
```

Here you still have a different version than in the development app and in the gemspec.
```suggestion
    "shakapacker": "~6.6.0",
```